### PR TITLE
Fix input realpower for Eaton ePDU and complete documentation

### PR DESCRIPTION
**Pull request by aquette**

### CODE REVIEW

**Comment by aquette** [comment]
damn, again forgotten to update the DMF part... update in progress

**Comment by jimklimov** [comment]
LGTM, except the missed DMF file update:

````
=== Are GitIgnores good after 'make all'? (should have no output below)
M scripts/DMF/dmfsnmp/eaton-pdu-marlin-mib.dmf
FATAL: There are changes in DMF files listed above - tracked sources should be updated!
````

Not sure why `Jenkinsfile` does not include or trigger this check, but that's a separate issue.

At least spellchecker is happy ;)

**Comment by jimklimov** [comment]
I'm sure I had a complex matrix-test `Jenkinsfile` somewhere, gotta undig it... but again, a separate issue.

File: drivers/eaton-pdu-marlin-mib.c [hunk 1]
```diff
@@ -582,7 +582,7 @@ static snmp_info_t eaton_marlin_mib[] = {
 		NULL, SU_FLAG_STATIC | SU_FLAG_OK | SU_TYPE_DAISY_1, NULL, NULL },
 	/* inputPowerCapacity.0.1 = INTEGER: 2300 */
 	/* FIXME: RFC on key name is needed when backporting to NUT upstream */
-	{ "input.power.nominal", 0, 1.0, ".1.3.6.1.4.1.534.6.6.7.3.5.1.9.%i.1",
+	{ "input.realpower.nominal", 0, 1.0, ".1.3.6.1.4.1.534.6.6.7.3.5.1.9.%i.1",
```
Comment (jimklimov) [inline drivers/eaton-pdu-marlin-mib.c]: So, update the `scripts/DMF/dmfsnmp/eaton-pdu-marlin-mib.dmf` with `power => realpower` change and bump the MIB version as appropriate. Or regenerate and use the result.

Comment (aquette) [reply drivers/eaton-pdu-marlin-mib.c]: gone the manual way, I've some issues with the regen that need audit

Comment (jimklimov) [reply drivers/eaton-pdu-marlin-mib.c]: Seems the MIB version is not updated (see the `mib2nut` tag in the end of file).

**Pull request merged**